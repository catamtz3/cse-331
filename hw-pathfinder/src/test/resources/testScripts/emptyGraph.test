CreateGraph emptyGraph

ListNodes emptyGraph

FindPath emptyGraph a b