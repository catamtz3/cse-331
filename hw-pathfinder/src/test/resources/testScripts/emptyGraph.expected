created graph emptyGraph

emptyGraph contains:

unknown node a
unknown node b